cttportfolio.css

```
@@ -3,168 +3,481 @@
-	width:33%;
-	height:33%;
+.project1 .project_text{
+	margin-left:50px;
 }
-.subprojectlink > *{
-	width:100%;
+.project2 .project_text{
+	margin-right:50px;
+}
+.project_main_links{
+	width:70%;
+	align-self:flex-end;
+	display:flex;
+	justify-content:space-between;
+	padding-bottom:2%;
+}
+.project_main_links .project_icon_size{
+	width:8%;
+}
+.pf_top button{
+	margin-top:80px;
+	margin-bottom:80px;
+}
+/*======================================== SUB PORTFOLIO SECTION ========================================*/
+.pf_sub{
+	display:flex;
+	justify-content:space-between;
+	height:500px;
+}
+.pf_sub > a {
+	width:30%;
+	height:100%;
+	text-decoration:none;	
+}
+.pf_sub_link_div{
 	height:100%;
 	display:flex;
 	flex-direction:column;
 	align-items:center;
 }
-.mainprojectlink > img{
+.pf_sub_link_div > img{
 	object-fit:cover;
+	height:100%;
+	max-width:100%;
 }
-a.portfoliolinks:hover{
-	border:1px solid black;	
+.pf_sub h6{
+	text-decoration:none;
+	font-weight:600;
+	font-size:2.0rem;
+	padding:5%;
 }
-
 /*======================================== CONTACT SECTION ========================================*/
 .contactme{
-	background-color:red;
+	display:flex;
+	align-items:center;
+	flex-direction:column;
+	margin-top:100px;
 }
 .contactmetext{
 	display:flex;
+	flex-direction:column;
 	justify-content:space-between;
-	padding:0 10%;
+	align-items:center;
+	width:100%;
+}
+.contactmetext > div{
+	margin:50px 0px;
+	padding:50px 50px;
+}
+.contact_div{
+	display:flex;
+	justify-content:space-between;
+	align-items:center;
+	width:80%;
+}
+.contact_div > *{
+	margin:0;
+}
+.contact_div h6{
+	font-size:2rem;
+}
+.contact_input{
+	width:50%;
+}
+.contact_input {
+	margin-top:50px;
+	display:flex;
+}
+.contact_input input{
+	margin-right:50px;
+	width:100%;
+	font-size:2rem;
+	padding-left:10px;
+	border-radius:10px;
+}
+.contact_input button{
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.0)
+}
+.contact_input button:hover{
+	border:1px solid rgba(248,248,255,0.6);
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.6)
+}
+.contact_div > h6{
+	font-size:2.0rem;
+}
+.contact_div > p{
+	font-size:2.5rem;
 }
 /*======================================== FOOTER SECTION ========================================*/
 footer{
-	background-color:#ffff66;
 	display:flex;
-	justify-content:space-between;
+	justify-content:center;
+	align-items:center;
 	padding:0 30%;
+	height:300px;
 }
-/*======================================== PROJECT PAGE SECTION ========================================*/
-.projectpageinfo{
+.footer_links{
+	max-width:100%;
+	margin:0 auto;
 	display:flex;
 	justify-content:space-between;
+	align-items:center;
+	height:30%;
+	width:60%;
+}
+.footer_icon_size{
+	width:33%;
+	height:100%;
+}
+.footer_icon_size > a > img{
+	display:block;
+	max-width:100%;
+	max-height:100%;
+	border-radius:50%;
+}
+/*======================================== PROJECT PAGE SECTION ========================================*/
+.pp_body h1, .pp_body h4{
+	font-size:4.0rem;
```

You don't need `.0` here.
```suggestion
	font-size:4rem;
```